core/common/src/main/java/alluxio/util/IdUtils.java

```diff
@@ -85,6 +86,14 @@ public static long createSessionId() {
     return getRandomNonNegativeLong();
   }
 
+  /**
+   * @return a session ID used internally in workers
```

Comment:
Since this is intended to only be used in the worker, should we move it to a utils class which is in the worker package?

Reply:
Done.